app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -135,6 +134,11 @@ public static DocumentEditFragment newInstance(File path, boolean pathIsFolder,
     private MarkorWebViewClient _webViewClient;
     private boolean _nextConvertToPrintMode = false;
     private long _loadModTime = 0;
+    private boolean _isTextChanged = true; // true forces first update
+    private Activity _activity;
+    private MenuItem _saveMenuItem;
+    private MenuItem _undoMenuItem;
+    private MenuItem _redoMenuItem;
```

Comment:
I had similar needs in the past - to access menu items any time. For that I've added at some point an accessor at GsFragmentBase

![grafik](https://user-images.githubusercontent.com/6735650/145416755-f8c59279-4fc0-40dd-b569-1d1e2abe565b.png)

this one can be used

Reply:
I thought of this, but we check view state _every time text is changed_ and calling `findViewById` every time might be a performance issue.

Reply:
I'd say it's not performance relevant (unlike syntax highlighter). If you want to keep it this way with storing it, I will check if I can slim it a bit still